src/shared/java/org/truffleruby/shared/Info.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved. This
+ * code is released under a tri EPL/GPL/LGPL license. You can use it,
+ * redistribute it and/or modify it under the terms of the:
+ *
+ * Eclipse Public License version 1.0
+ * GNU General Public License version 2
+ * GNU Lesser General Public License version 2.1
+ */
+
+package org.truffleruby.shared;
+
+public class Info {
```

Comment:
`Info` is very generic, maybe `RubyConstants` or `VersionConstants` or `SharedConstants`?

Reply:
I not opposed to renaming, but it does not contain just constants so I think constants should not be in the name.

Reply:
Well it's constants and a couple helper methods to concatenate constants and build `RUBY_` constants basically :)
Another idea: `EngineInfo` since it's mostly engine-related information.
Could also be `TruffleRuby` maybe, it looks nice as `TruffleRuby.getVersionString()`.